app/steps/ui/summary/template.html

```diff
@@ -10,7 +10,7 @@ <h1 class="form-title heading-large">{{ content.Summary.heading }}</h1>
 
 {% block form_content %}
     <div class="form-group">
-        <p>{{ content.Summary.checkCarefully }}</p>
+        <p id="main-heading-content">{{ content.Summary.checkCarefully }}</p>
```

Comment:
Do we need this id here?

Reply:
yes it identifies the main paragraph content, other section headings are identified via their classes. This is a direct copy from probate.